ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -491,7 +518,11 @@ void RSkComponentTextInput::resignFromEditingMode() {
       drawAndSubmit();
     }
   }
-  RNS_LOG_DEBUG("[requestForEditingMode] END");
+  RNS_LOG_DEBUG("[RSkComponentTextInput][requestForEditingMode] *** END ***");
```

Comment:
do we need to handle the qyeye clearing and reset the keyrepeat flags in blur command as well